calculator/main.py

```diff
@@ -26,8 +26,18 @@ def perform_operation(calc, operation, params):
     :param params: Tuple containing the list of nums to operate with.
                    ie: (1, 2, 3, 4.5, -2)
     """
-    pass
-
+    # perform operation
+    for i in params:
```

Comment:
might want to put something more descriptive than `i`. `for arg in params` or something, perhaps? so that future readers of your code (may include yourself) can see at a glance what each individual element is.